FBC/DebtAndEquities/CreditEvents.rdf

```diff
@@ -221,30 +227,24 @@
 	</owl:Class>
 	
 	<owl:ObjectProperty rdf:about="&fibo-fbc-dae-cre;hasDefaultThresholdAmount">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-acc-cur;hasMonetaryAmount"/>
 		<rdfs:label xml:lang="en">has default threshold amount</rdfs:label>
 		<rdfs:domain rdf:resource="&fibo-fbc-dae-cre;EntitySpecificCreditEvent"/>
 		<rdfs:range rdf:resource="&fibo-fnd-acc-cur;MonetaryAmount"/>
 		<skos:definition xml:lang="en">specifies an amount of money that triggers a repudiation/moratorium or restructuring event</skos:definition>
 	</owl:ObjectProperty>
 	
-	<owl:DatatypeProperty rdf:about="&fibo-fbc-dae-cre;hasExtendableGracePeriod">
-		<rdfs:label xml:lang="en">has extendable grace period</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-fbc-dae-cre;FailureToPay"/>
-		<rdfs:range rdf:resource="&xsd;boolean"/>
-		<skos:definition xml:lang="en">indicates whether or not the grace period may be extended, which may be jurisdiction specific</skos:definition>
-	</owl:DatatypeProperty>
-	
 	<owl:ObjectProperty rdf:about="&fibo-fbc-dae-cre;hasGracePeriod">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-dt-fd;hasDatePeriod"/>
```

Comment:
as for hasThreshold this should be a property of the Contract

Reply:
It is now.